handlers/handlers.go

```diff
@@ -115,15 +115,13 @@ func HandleUserApiRequest(
 			return
 		}
 
-		user1 := models.User{"Adrian"}
-		user2 := models.User{"Evan"}
-
-		usersById := map[models.UserId]models.User{
-			1: user1,
-			2: user2,
+		usersById, err := env.Db.GetAllUsersById()
+		if err != nil {
+			http.Error(responseWriter, err.Error(), http.StatusInternalServerError)
+			return
 		}
 
-		usersByIdJson, err := json.Marshal(usersById)
+		usersByIdJson, err := usersById.ToJson()
```

Comment:
I commented in the last pr about ToJson. If you let usersById implement the json.Marshaller interface you can do  `json.NewEncoder(responseWriter).Encode(usersById)`